src/main/java/me/xhyrom/spawnergenz/SpawnerGenz.java

```diff
@@ -35,7 +47,16 @@ public void onLoad() {
     public void onEnable() {
         CommandAPI.onEnable();
         saveDefaultConfig();
-
+        if (!setupVault()) {
```

Comment:
We can move this things into hooks/Hooks instead of flooding main class with this stuff.